### PR TITLE
Task #152: evaluation bar

### PR DESCRIPTION
**Pull request by Thalamicks**
in this branch I added the following features:

- New frontend API call: _getWinningPercentage_
- New component: _VictoryBar.jsx_  (implements the evaluation bar functionality)

### CODE REVIEW

**Review by thorsjur — APPROVED** [review]
Looks very good! The only thing i noticed is that it tries to display some non-numbers when a player has won, which isn't really a big deal:

image

Also check out the single code comment, else it looks great

File: frontend/src/components/victoryBar/VictoryBar.jsx [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+        </Typography>
+      </Box>
+    </Box>
+  );
+};
+
+VictoryBar.propTypes = {
+  whitePercentage: PropTypes.number.isRequired,
+  blackPercentage: PropTypes.number.isRequired,
+};
```
Comment (thorsjur) [inline frontend/src/components/victoryBar/VictoryBar.jsx:68]: If i'm not mistaken this isn't necessary, as the component doesn't take any props at the moment.

Comment (Thalamicks) [reply frontend/src/components/victoryBar/VictoryBar.jsx]: Yea it was a typo thanks

**Comment by Thalamicks** [comment]
I just fixed some bugs in the last commit and noticed that there is a problem with the value returned by the _get_winning_percentage_ API when a player wins. This is why the bar displays incorrect values when one of the two has a 100% chance of winning

**Comment by thorsjur** [comment]
Looks good, so feel free to merge when ready 👍

**Pull request merged**